diff.txt

```diff
@@ -0,0 +1,879 @@
+directory 'bndtools/'
+directory 'bndtools/'
+directory 'bndtools/'
+
+
+blade.authenticator.shiro-1.0.0.jar
+*******************************************************
+
+
+blade.authfailure-1.0.0.jar
+*******************************************************
+
+
+blade.configurationaction-1.0.0.jar
+*******************************************************
+CHANGED    MANIFEST   <manifest>
+ ADDED      HEADER     Require-Capability:osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://java.sun.com/portlet_2_0))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/aui))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/portlet))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/theme))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/ui))",osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.7))"
+ REMOVED    HEADER     Require-Capability:osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://java.sun.com/portlet_2_0))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/aui))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/portlet))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/theme))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/ui))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/aui))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/portlet))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/theme))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/ui))",osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.7))"
+CHANGED    RESOURCES  <resources>
+ ADDED      RESOURCE   OSGI-INF/metatype/com.liferay.blade.samples.configurationaction.MessageDisplayConfiguration.xml
+
+
+blade.corejsphook-1.0.0.jar
+*******************************************************
+CHANGED    MANIFEST   <manifest>
+ CHANGED    HEADER     Import-Package
+  ADDED      CLAUSE     javax.servlet
+  ADDED      CLAUSE     javax.servlet.http
+
+
+blade.friendlyurl-1.0.0.jar
+*******************************************************
+
+
+blade.gogo-1.0.0.jar
+*******************************************************
+
+
+blade.hook.jsp-1.0.0.jar
+*******************************************************
+CHANGED    MANIFEST   <manifest>
+ ADDED      HEADER     Import-Package
+
+
+blade.hook.resourcebundle-1.0.0.jar
+*******************************************************
+
+
+blade.indexerpostprocessor-1.0.0.jar
+*******************************************************
+
+
+blade.jsf.portlet.ds-1.0.0.jar
+*******************************************************
+CHANGED    MANIFEST   <manifest>
+ CHANGED    HEADER     Import-Package
+  CHANGED    CLAUSE     javax.servlet
+   ADDED      PARAMETER  version:[3.0,4)
```

Comment:
So it appears that in the liferay-gradle we are getting stricter import ranges of [3.0,4) i think this may be because of some different inputs to bnd builder between liferay-gradle and others.